test/cadet_web/controllers/auth_controller_test.exs

```diff
@@ -1,89 +1,71 @@
 defmodule CadetWeb.AuthControllerTest do
+  @moduledoc """
+  Some tests in this module use pre-recorded HTTP responses saved by ExVCR.
+  this allows testing without the use of actual external IVLE API calls.
+
+  In the case that you need to change the recorded responses, you will need
+  to set the two environment variables IVLE_KEY (used as a module attribute in
+  `Cadet.Accounts.IVLE`) and TOKEN (used here). Don't forget to delete the
+  cassette files, otherwise ExVCR will not override the cassettes.
+  """
+
   use CadetWeb.ConnCase
+  use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
 
   import Cadet.Factory
 
   alias CadetWeb.AuthController
   alias Cadet.Auth.Guardian
 
+  @token String.replace(inspect(System.get_env("TOKEN")), ~s("), "")
```

Comment:
Not sure what this does, why not a simple `System.get_env("TOKEN")`? Also what is this `TOKEN` environment variable?

Reply:
The token is the authentication token from ivle for the user. Added a brief description in the module docs of tests using `@token`, and improved the readability with 7075f93.